spec/active_record/locking_extensions_spec.rb

```diff
@@ -1,7 +1,6 @@
 # encoding: utf-8
-require 'spec_helper'
```

Comment:
Not one for reading the PR description are you :wink:
It's specified in the `.rspec` file.